_ext/pipeline.rb

```diff
@@ -34,6 +33,8 @@
   helper Awestruct::Extensions::GoogleAnalytics
 
   extension Awestruct::Extensions::Indexifier.new
+  extension Awestruct::Extensions::Sitemap.new
+  extension Awestruct::Extensions::Tagger.new( :posts, '/news/index', '/news/tags', :per_page=>5)
```

Comment:
Moved Tagger extension after the Sitemap extension. Little hack but I didn't find any other way how to exclude all files under /news/tags/ during sitemap generation.